### PR TITLE
GUM-40: Add quiz 

### PR DESCRIPTION
**Pull request by katyakostikova**

### CODE REVIEW

File: src/screens/quiz/components/quiz-item/quiz-item.tsx
Comment (AleksandrTermenzhy) [inline src/screens/quiz/components/quiz-item/quiz-item.tsx:20]: Do we need `'0'` here or empty string can fit too?

File: src/screens/quiz/components/quiz-item/quiz-item.tsx
Comment (AleksandrTermenzhy) [inline src/screens/quiz/components/quiz-item/quiz-item.tsx]:
```suggestion
              <RadioButton value={index?.toString()} />
```

File: src/screens/quiz/quiz.tsx
Comment (AleksandrTermenzhy) [inline src/screens/quiz/quiz.tsx]:
```suggestion
  const handleAnswerChange = (
```

File: src/screens/quiz/components/quiz-item/quiz-item.tsx
Comment (AleksandrTermenzhy) [inline src/screens/quiz/components/quiz-item/quiz-item.tsx]:
```suggestion
  onAnswerChange: (
```

**Review by AleksandrTermenzhy — APPROVED** [review]

File: src/screens/profile/components/quiz-info/styles.ts
Comment (R-Mielamud) [inline src/screens/profile/components/quiz-info/styles.ts]: Please wait until #48 is merged and replace with `useStyles` hook.

File: src/screens/profile/profile.tsx
Comment (R-Mielamud) [inline src/screens/profile/profile.tsx]: Use one selector and form an object inside it.

File: src/screens/quiz/styles.ts
Comment (R-Mielamud) [inline src/screens/quiz/styles.ts]: Also replace with `useStyles`/`useColor` when #48 is merged

File: src/store/quiz/slice.ts
Comment (R-Mielamud) [inline src/store/quiz/slice.ts]: Can you simplify this? As I know, all ids are UUIDs

**Review by R-Mielamud — APPROVED** [review]

**Pull request merged**
